pkg/accountmanager/client.go

```diff
@@ -11,48 +11,43 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 )
 
+//go:generate mockery -name clienter
 type clienter interface {
-	Config(roleArn *arn.ARN) *aws.Config
+	Config(roleArn *arn.ARN, roleSessionName string) *aws.Config
```

Comment:
We have a business requirement for session names to match our principal IDs (it allows us to track CloudTrail events back to actual users)